scarb-metadata/src/lib.rs

```diff
@@ -208,6 +208,9 @@ pub struct PackageMetadata {
     /// Package version as given in `Scarb.toml`.
     pub version: Version,
 
+    /// Package edition as given in `Scarb.toml`.
+    pub edition: String,
```

Comment:
Adding new fields without serde falling back is a breaking change here. I think we should use `Option<String>` here and skip serializing if field is `None`. Also don't forget about deserializing from older Scarbs